I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 396, in script
    jump bad
ScriptError: could not find label 'bad'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 396, in script
    jump bad
  File "C:\Users\edmun\OneDrive\Desktop\renpy-7.3.5-sdk\renpy\ast.py", line 1697, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Users\edmun\OneDrive\Desktop\renpy-7.3.5-sdk\renpy\script.py", line 898, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'bad'.

Windows-8-6.2.9200
Ren'Py 7.3.5.606
Unremarkable Everyday 1.0
Thu May 21 14:44:58 2020
